src/main/java/mate/academy/dao/impl/MovieDaoImpl.java

```diff
@@ -45,6 +46,12 @@ public Optional<Movie> get(Long id) {
 
     @Override
     public List<Movie> getAll() {
-        return null;
+        try (Session session = HibernateUtil.getSessionFactory().openSession()) {
+            Query<Movie> getAllMoviesQuery = session.createQuery(
+                    "from Movie", Movie.class);
```

Comment:
please keep HQL keywords uppercase for better readability
```suggestion
                    "FROM Movie", Movie.class);
```